### DISCUSSION TITLE
PWA Desktop Icon remains Quasar logo

### DISCUSSION THREAD

**Discussion by rconstantine** [quasarframework/quasar#10811] [CLI - PWA mode]
Hey folks, I have Icon Genie installed and it produced all of the icons based on the image I provided. I added the required lines to my **index.template.html** file and removed the lines that were previously there which referenced icons I no longer have.

I have gone all through my src and public folders and see no Quasar icons anywhere. The only ones that exist are in the node_modules @quasar package.

When I view the app as a web page, the favicon shows up just fine in the tab, in the favorites, etc.

But when I install it, the icon that is put on the desktop (Win 10) is the Quasar logo. I didn't see anything in either the PWA or Icon Genie docs to troubleshoot that. Did I just miss something?

**Comment by ontwikkelfabriek** [comment]
@rconstantine 

I would first, if possible try to install the PWA on another computer/VM to see if the issue is resolved.

It is probably browser cache, even more likely when for an installed PWA. An installed PWA will update itself but usually only once a day, this van be instant on deploying a new version, but you have to do some coding in the correct hook in the service worker.  

Try deleting the website app cache. For example in Chrome -> dev tools -> Application -> Storage -> Clear site data ( select the appropriate checkboxes).
